samples/cellular/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -91,12 +113,22 @@ static int init_leds(void)
 	return err;
 }
 
-static void button_handler(uint32_t button_states, uint32_t has_changed)
+static bool app_event_handler(const struct app_event_header *aeh)
 {
-	if (has_changed & button_states & BIT(BTN_NUM - 1)) {
+	struct button_event *event = is_button_event(aeh) ? cast_button_event(aeh) : NULL;
+
+	if (!event) {
+		return false;
+	} else if (!event->pressed) {
+		return true;
+	}
+
+	if (event->key_id == (uint16_t)(BTN_NUM - 1)) {
```

Comment:
you could move `LOG_DBG("Button %d pressed", BTN_NUM);` before the key_id check, so there is a log message for any button press?